Makefile

```diff
@@ -35,3 +35,11 @@ local-database: clean_test_db
 .PHONY: local-backend
 local-backend:
 	$(MAKE) local-server -C ./backend/chalice/api_server
+
+.PHONY: smoke-test-with-local-backend
+smoke-test-with-local-backend:
```

Comment:
This allows people to run `make smoke-test-with-local-backend` in the root directory